src/test/tests-setup.js

```diff
@@ -43,7 +43,9 @@ const config = {
 
 global.DebuggerConfig = config;
 global.L10N = require("devtools-launchpad").L10N;
-global.L10N.setBundle(readFileSync("./assets/panel/debugger.properties"));
+global.L10N.setBundle(
+  readFileSync(path.join(__dirname, "../../assets/panel/debugger.properties"))
```

Comment:
yikes

Reply:
I just happened to run Jest from a subdirectory and it wasn't happy :P

Reply:
oh ha - hadnt thought of that